docs/bot/configuration.md

```diff
@@ -43,3 +43,39 @@ These are the keys you can specify:
   "contributors": []
 }
 ```
+
+#### Moving the All Contributors table into a different file
+
+As you know, the default file for the contributors table is `README.md`. But if your project has tons of contributors, it doesn't make sense to include the All Contributors table in the `README.md` file. Instead, you can use a different file to do so.
+
+Here are the steps that may help you in achieving that. For the sake of this tutorial, the different file that we'll use is going to be named `CONTRIBUTORS.md`.
+
+1. Install the bot. See the [Installation](installation) page for instructions.
+2. Add your first contributor. See the [Bot Usage](usage) page for instructions.
+3. Create the `CONTRIBUTORS.md` file.
+4. Copy the code from the Contributors section in the `README.md` file, starting at the `ALL-CONTRIBUTORS-LIST:START` line, and ending at the `ALL-CONTRIBUTORS-LIST:END` line.
+
+    Example:
+    ```html
```

Comment:
Nit: You don't need that indentation before the triple tildes (same goes for the lines below).

Reply:
So, all the code blocks doesn't need to be indented isn't it?

Reply:
![image](https://user-images.githubusercontent.com/66242389/148619228-eaa3ad1f-f9a7-443c-9982-c1d046ef3e94.png)
after remove the indentations, it looks something like this

Reply:
Yeah, the indentation is needed to make the code block (and the "Example" paragraph) a child of the list item. Indenting by 3 characters would be sufficient, though.

Reply:
@waldyrious I guess, I'll leave it with the current indentation.

Reply:
@waldyrious I've changed it from 4 spaces to 3. Please take a look at my latest commit 😁 Thanks

Reply:
Hmm, you seem to have introduced tab-based indentation in the code blocks. Please re-check if that's what you intended.

Reply:
I actually doesn't know how to differentiate between tab-based and space-based since I only use tab for indentation. May you take a look at my current commit? And if you're using VS Code, can you share with me how to differentiate those 2 types of indentations? Thanks.